docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,333 @@ Review important information about the {kib} 8.x releases.
+Observability::
+* Enable burn rate alert by default during creation via UI ({kibana-pull}176317[#176317]).
+* Implements Bedrock support for the Claude models ({kibana-pull}176191[#176191]).
+* Adds link for AI Assistant in Observability left hand navigation ({kibana-pull}176144[#176144]).
+* Handle token limit error message improvement ({kibana-pull}175871[#175871]).
+* Corrects common {esql} mistakes ({kibana-pull}175520[#175520]).
+* Adds resource column with tooltip ({kibana-pull}175287[#175287]).
+* Adds alert fields table to Observability flyout ({kibana-pull}174685[#174685]).
+Platform::
+* Adds an option to disable APM user redaction ({kibana-pull}176566[#176566]).
+Reporting::
+* Updated CSV export to insert error messages into the contents if the export results in an empty file due to an error ({kibana-pull}175852[#175852]).
+* Adds setting to use PIT or Scroll API ({kibana-pull}174980[#174980]).
+Security::
+* Default value of `server.securityResponseHeaders.referrerPolicy` changed to `strict-origin-when-cross-origin` ({kibana-pull}177559[#177559]).
+* Adds the asset criticality UI ({kibana-pull}173512[#173512]).
+* Show Asset Criticality in Risk Input flyout ({kibana-pull}174638[#174638]).
+* Restrict management page based on user privileges ({kibana-pull}173178[#173178]).
+* Merge risk input and asset integration expandable panels ({kibana-pull}173028[#173028]).
+* Require full risk engine privileges to enable, disable and init the risk engine ({kibana-pull}172932[#172932]).
+* Adds server side validation for uploaded file types ({kibana-pull}173960[#173960]).
+Sharing::
+* Show 'View details' UI action to open clusters inspector tab when request fails ({kibana-pull}172971[#172971]).
+Unified Search::
+* Adds 'greater than or equals to' and 'less than' options to filter options for date ranges and numbers ({kibana-pull}174283[#174283]).
+* Adds a one minute option to the date picker ({kibana-pull}172944[#172944]).
+* Adds multiple improvements to saved query management ({kibana-pull}170599[#170599]).
+
+[float]
+[[fixes-v8.13.0]]
+=== Bug Fixes
+Alerting::
+* Fixes Elasticsearch query rule with KQL evaluation matched document count ({kibana-pull}176620[#176620]).
+* Fixes alerts not being visible when number of alerts are more than max alert limit ({kibana-pull}178019[#178019]).
+* Fixes `ruleClient.getAlertState` error when a task is no longer available ({kibana-pull}177077[#177077]).
+* Fixes AlertsTable sorting with inactive columns and default sort direction ({kibana-pull}176172[#176172]).
+APM::
+* Fixes an infinite loop caused by matching child and parent IDs ({kibana-pull}177914[#177914]).
+* Fixes inconsistencies on Service Overview page ({kibana-pull}176293[#176293]).
+* Fixes occurrences cut off value to be fully visible ({kibana-pull}175307[#175307]).
+Cases::
+* Update UI to use custom field internal API ({kibana-pull}176574[#176574]).
```

Comment:
I noticed that we forgot to add a release note section to this PR https://github.com/elastic/kibana/pull/176574. I updated it. It should be "Fix a bug where If there are required custom fields on a case whose values are empty it is not possible to update any of those fields.".